apps/web/src/components/icons/s3-icon.tsx

```
@@ -0,0 +1,10 @@
+export default function S3Icon() {
+	return (
+		<svg viewBox="0 0 24 24" fill="currentColor" className="h-5 w-5">
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)